### DISCUSSION TITLE
stock, cant run latest sherlock

### DISCUSSION THREAD

**Discussion by DrCFC** [sherlock-project/sherlock#2365] [Q&A] [answered]
┌──(kali㉿kali)-[~/Desktop/sherlock]
└─$ sherlock_project
                                                                                                  
┌──(kali㉿kali)-[~/Desktop/sherlock/sherlock_project]
└─$ ls
__init__.py  __main__.py  notify.py  py.typed  resources  result.py  sherlock.py  sites.py
                                                                                                  
┌──(kali㉿kali)-[~/Desktop/sherlock/sherlock_project]
└─$ python sherlock.py
Did you run Sherlock with `python3 sherlock/sherlock.py ...`?
This is an outdated method. Please see https://sherlockproject.xyz/installation for up to date instructions.

website not working, cant see latest methods to run sherlock

**Comment by bytexenon** [comment, marked as answer]
Hi,

If you don't need to modify Sherlock itself, it's recommended to install it using `pipx`:

```sh
pipx install sherlock-project
```

You don't need to install it from source from Github. 

And, since the site is down for you, here are some screenshots of that page:

![Screenshot 1](https://github.com/user-attachments/assets/bc31859c-098c-4769-9f2f-dff27a30808e)
![Screenshot 2](https://github.com/user-attachments/assets/4547027f-3d56-4d96-bfb6-f5311998c354)
![Screenshot 3](https://github.com/user-attachments/assets/eb1eeb58-c4e7-4139-aaae-dec1861f52ff)
